app/assets/javascripts/transactions.js

```diff
@@ -1,4 +1,5 @@
 $(function () {
+  $('#transaction_search').hide();
```

Comment:
@yunixon we need to hide filters only if we have not used it. If we filtered transactions - we need to keep Filters showed.